api/applications/applicationRouter.js

```diff
@@ -353,27 +361,51 @@ router.post(
  *        $ref: '#/components/responses/UnauthorizedError'
  */
 
-// registers a user with okta and approves their application ticket
-
-router.put(
-  '/approve/:id',
-  authRequired,
-  adminRequired,
-  checkApplicationExists,
-  findProfile,
-  registerOktaUser,
-  (req, res, next) => {
-    const application_id = req.application.application_id;
+// Author: Christwide Oscar
+// Finding applicant and differentiating between mentors and mentees. Setting their application status to 'approved' from pending
+router.post('/approve/:profile_id', (req) => {
+  const { profile_id, low_income } = req.body;
+  if (typeof low_income !== undefined) {
+    axios
+      .post(`${baseURL}/update/mentee/${profile_id}`, {
+        validate_status: 'approved',
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  } else {
+    axios
+      .post(`${baseURL}/update/mentor/${profile_id}`, {
+        validate_status: 'approved',
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }
+});
 
-    Application.updateTicket(application_id, { approved: true })
-      .then(() => {
-        res.status(200).json({
-          tempPassword: req.tempPassword,
-        });
+// Author: Farhaan Nishtar
+// Finding applicant and setting them to rejected
+router.post('/reject/:profile_id', (req) => {
+  const { profile_id, low_income } = req.body;
+  if (typeof low_income !== undefined) {
+    axios
+      .post(`${baseURL}/update/mentee/${profile_id}`, {
+        validate_status: 'rejected',
       })
-      .catch(next);
+      .catch((err) => {
+        console.log(err);
```

Comment:
We'll need to raise an issue for this FE for error handling--there are a number of components that are using console.logs instead of setting state/sending a message per the response. Just an fyi for next steps.